lib/demos/cupertino/cupertino_context_menu_demo.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/gallery_localizations.dart';
+
+// BEGIN cupertinoContextMenuDemo
+
+class CupertinoContextMenuDemo extends StatelessWidget {
+  const CupertinoContextMenuDemo();
+
+  @override
+  Widget build(BuildContext context) {
+    final galleryLocalizations = GalleryLocalizations.of(context);
+    return CupertinoPageScaffold(
+      navigationBar: CupertinoNavigationBar(
+        middle: Text(galleryLocalizations.demoCupertinoContextMenuTitle),
```

Comment:
Add `automaticallyImplyLeading: false` so the back button doesn't appear

Reply:
ok